src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PUblishView.cs

```diff
@@ -91,6 +98,8 @@ private void PopulateView()
 
                             owners = owners.Union(organizationLogins).ToArray();
                         });
+                    }).Finally(task => {
+                        isBusy = false;
```

Comment:
aaaah yes we're not in the new task system here. bummer. Ok we can't use `Finally` here, it's on the wrong thread. Let's set `isBusy = false` in all the return points inside the callbacks for now and then talk about how this API should actually look like separately.